apps/rest/run_evadb.py

```diff
@@ -0,0 +1,86 @@
+from flask import Flask
+from flask_restful import Api, Resource
+import evadb
+
+app = Flask(__name__)
+api = Api(app)
+
+cursor = evadb.connect().cursor()
+
+class DatabaseResource(Resource):
+    def get(self,db_name):
+
+        # Create a database
+        query = """ CREATE DATABASE """ + db_name + """
+            WITH ENGINE = 'postgres',
+            PARAMETERS = {
+                "user": "eva",
+                "password": "password",
+                "host": "localhost",
+                "port": "5000",
+                "database":""" + db_name + """
+            }; """
+        
+        res = cursor.query(query).df()
+        return {"msg" : "Created Database" , "response" : res.to_json()}
+
+class TableResource(Resource):
+    def get(self, name):
+        query = """ 
+            CREATE TABLE IF NOT EXISTS """ + name + """ (
+                id INTEGER UNIQUE,
+                label TEXT(30)
+                    ); 
+        """
+        res = cursor.query(query).df()
+        return {"msg" : "Created table ", "response" : res.to_json() }
+    
+    def post(self,name):
+        query = """
+                INSERT INTO """+ name + """ (id, label)
+                VALUES (
+                2,
+                '7019462-Eva'
+                );
+        """
+        res = cursor.query(query).df()
+        return {"msg" : "Entry added to database" , "response" : res.to_json()}
+    
+    def delete(self,name):
+        query = """
+                DROP table """+ name +""";
+            """
+        res = cursor.query(query).df()
+        return {"response" : res.to_json()}
+
+
+
+api.add_resource(DatabaseResource, "/db/<string:name>")
+api.add_resource(TableResource, "/table/<string:name>")
+
+
+# GET UDFS
+@app.route('/udfs', methods = ['GET'])
+def udfs():
+    res = cursor.query("SHOW UDFS;").df()
+    return {"response" : res.to_json()}   
+
+#GET FUNCTIONS
+@app.route('/func', methods = ['GET'])
+def func():
+    res = cursor.query("SHOW FUNCTIONS;").df()
+    return {"response" : res.to_json()}     
+
+@app.route('/table_data/<name>',methods = ['GET'])
+def get_data(name):
+    query = "Select * from" + name + ";"
+    res = cursor.query(query).df() #Show 
+    return {"response" : res.to_json()}
+
+@app.route('/query/<query>',methods = ['GET'])
+def query_from_db(query):
```

Comment:
I think all we need is the query. The rest is not necessary. However, we need to optimize the query endpoint. Some I can think of now:
1. we may need a queue to serialize concurrent queries. @jiashenC has done some setup similar for slackbot. You can refer to that. 
2. We may also need the response to be streaming, considering we have a very large table as the result. 

Other than query, we can also have a upload endpoint for users to upload their csvs, pdfs, videos.

Reply:
Hey Andy. I added the upload endpoint. Could you take a look in my latest commit?